packages/@lwc/ssr-compiler/src/compile-js/generate-markup.ts

```diff
@@ -41,7 +41,11 @@ const bGenerateMarkup = esTemplate<ExportNamedDeclaration>`
 `;
 
 const bInsertFallbackTmplImport = esTemplate<ImportDeclaration>`
-    import { fallbackTmpl as __fallbackTmpl, renderAttrs as __renderAttrs } from '@lwc/ssr-runtime';
+    import {
+        fallbackTmpl as __fallbackTmpl,
+        renderAttrs as __renderAttrs,
+        SYMBOL__SET_INTERNALS as __SYMBOL__SET_INTERNALS,
```

Comment:
This rename seems unnecessary to me, especially since someone could still hack around it by using the `__SYMBOL__SET_INTERNALS` variable. I'd say let's just keep the name as-is, and add some AST checks to throw an error if consumers try to use the floating global variable (or confirm that Rollup shadows the variable correctly – not sure what happens).

Reply:
My understanding is that the local variable is just an attempt to minimize namespace collisions, and nothing more. We already have an allow list in the compiler that prevents direct importing of unsupported APIs. Does this address your concern, or did you mean something else?

Reply:
I mean, can someone just do

```js
connectedCallback() {
  goToTown(__SYMBOL__SET_INTERNALS)
}
```

Now that I think about it though, I guess not, since their `connectedCallback` lives in another file, and Rollup is smart about shadowing when it bundles. So I think we are fine!